res/css/_sc/feature-improvement/consistency__TextEvent/index.pcss

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2024 Suguru Hirahara
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/* Fix upstream's style inconsistencies among text events and mx_DecryptionFailureBody */
+/* See https://github.com/SchildiChat/schildichat-desktop/issues/240 */
+
+@import "./views/rooms/_EventTile.pcss";
+@import "./views/messages/_DecryptionFailureBody.pcss";
+@import "./views/messages/_DisambiguatedProfile.pcss";
+
+@define-mixin consistency__TextEvent {
+    /* Applied to IRC, modern layout, and the right panel. Not applied to the
+       rest of the layouts. Remove the nested :where() if it is applied to them
+       as well. */
+    :where(.mx_RoomView_body:where([data-layout="irc"], [data-layout="group"]), .mx_RightPanel) {
```

Comment:
Takes advantage of `:where()` which has no specificity. See https://web.dev/articles/css-is-and-where for its merit.